test/db.test.js

```diff
@@ -1,79 +0,0 @@
-const path = require('path');
```

Comment:
this file was added to test ssl connection just recently why it is removed ?

Reply:
Good question.
I deleted it temporarily because I'll have to figure out how to test that part.

The problem here is — `ioredis` gets created using a constructor and not using a function such as `Redis.createClient()`

`sinon`, the stub library we're using, does not support spies on `constructors`.

I deleted this test just to verify it was working correctly. I'll now figure out another way to test this part.

Reply:
👍